src/components/Swap/Output.tsx

```diff
@@ -98,7 +101,7 @@ export default function Output({ disabled, focused, children }: PropsWithChildre
         </TokenInput>
         <Rule />
         {children}
-        <BrandedFooter />
+        {!disableBranding && <BrandedFooter />}
```

Comment:
nit: because this is a double negative, it may be more readable as

    {disableBranding ? null : <BrandedFooter />}

Reply:
the negative in the prop is so gross to work with, but it really feels like the right name 😱